cfunits/units.py

```diff
@@ -2213,14 +2214,20 @@ def conform(cls, x, from_units, to_units, inplace=False):
             # --------------------------------------------------------
             if inplace:
                 if x.dtype.kind == "i":
-                    if x.dtype.char == "i":
+                    # Here it is only checked for signed integer:
+                    # Are unsigned integers not possible?
+                    if x.dtype in [numpy.int32]:
```

Comment:
Sorry I forgot to submit one comment I meant to attach to the previous review: instead of checking for presence in a one-item list, it is probably simpler and clearer to use:

```suggestion
                    if x.dtype == numpy.int32:
```

unless there is some reason you know that it isn't a good idea to do that? (The same applies to the same form of statement used in the following lines.)

Reply:
I was thinking that we should do the same for many different dtypes. Therefore are started using a list.

But anyway, due to my comments to the issue #31 I provide a better code without importing numpy.

Reply:
That makes sense. No problem.